msal/application.py

```diff
@@ -69,8 +69,11 @@ def __init__(
                 {
                     "private_key": "...-----BEGIN PRIVATE KEY-----...",
                     "thumbprint": "A1B2C3D4E5F6...",
+                    "public_certificate": "...-----BEGIN CERTIFICATE-----..." (Only to be sent when using Subject Name Issuer Authentication)
```

Comment:
Given that we will have a dedicated paragraph to talk about this, I think we can replace that long "(Only to be sent ...)" part with a short message, something like "(Optional. See below.)".